### DISCUSSION TITLE
Getting an error when attempting to export mesh

### DISCUSSION THREAD

**Discussion by ArmaniG-W** [JoseConseco/GoB#444] [Q&A]
Hello,
I am having trouble exporting a mesh, I get an error. Probably user error but any help would be appreciated. I'm using Blender version 3.5.0 and trying to export to Zbrush version 2021.6.6

![image](https://github.com/JoseConseco/GoB/assets/103336414/77524d2e-bd06-4af0-a4a5-036dc408680d)

**Comment by kromar** [comment]
Hi @ArmaniG-W , can you check what version of GoB you are using?

**Reply by ArmaniG-W** [reply]
Sorry @kromar, thanks for responding. Turns out it was human error. I believe I was downloading the master instead of the newer releases
